Add issue about loading a lot of tables (#3975)
airbytehq/airbyte: docs/faq/data-loading.md

## **Why don’t I see any data in my destination yet?**

It can take a while for Airbyte to load data into your destination. Some sources have restrictive API limits which constrain how much 
data we can sync in a given time. Large amounts of data in your source can also make the initial sync take longer. You can check your
sync status in your connection detail page that you can access through the destination detail page or the source one.

## **What happens if a sync fails?**

You won't loose data when a sync fails, however, no data will be added or updated in your destination.

Airbyte will automatically attempt to replicate data 3 times. You can see and export the logs for those attempts in the connection 
detail page. You can access this page through the Source or Destination detail page.

> In the future, you will be able to configure a notification \(email, Slack...\) when a sync fails, with an option to create a 

Reviewer: This is actually live

Author: Thanks @michel-tricot, also I created https://github.com/airbytehq/airbyte/issues/3976 to made a small doc/tutorial to bring more visibility about this feature.

## **Why don’t I see any data in my destination yet?**

It can take a while for Airbyte to load data into your destination. Some sources have restrictive API limits which constrain how much 
data we can sync in a given time. Large amounts of data in your source can also make the initial sync take longer. You can check your
sync status in your connection detail page that you can access through the destination detail page or the source one.

## **What happens if a sync fails?**

You won't loose data when a sync fails, however, no data will be added or updated in your destination.

Airbyte will automatically attempt to replicate data 3 times. You can see and export the logs for those attempts in the connection 
detail page. You can access this page through the Source or Destination detail page.

> You can configure a Slack webhook to warn you when a sync failed.

Reviewer 2: ```suggestion
A Slack webhook can be configured for sync failures.
```

## **Do you support change data capture \(CDC\) or logical replication for databases?**

Airbyte currently supports [CDC for Postgres and Mysql](../understanding-airbyte/cdc.md). Airbyte is adding support for a few other 
databases you can check in the roadmap.

## Using incremental sync, is it possible to add more fields when some new columns are added to a source table, or when a new table is added?

For the moment, incremental sync doesn't support schema changes, so you would need to perform a full refresh whenever that happens.
Here’s a related [Github issue](https://github.com/airbytehq/airbyte/issues/1601).

## There is a limit of how many tables one connection can handle?

> Yes, for more than 6000 thousand tables could be a problem to load the information on UI.

Reviewer 2: link this to the new-connection section? centralising this makes it easier to maintain.

something like `Yes. Please see the this section for more information.'
